<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmDepto</class>
 <widget class="QMainWindow" name="FrmDepto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Departamentos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="abas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabCadastro">
     <attribute name="icon">
      <iconset resource="recursos.qrc">
       <normaloff>:/depto/COPY.BMP</normaloff>:/depto/COPY.BMP</iconset>
     </attribute>
     <attribute name="title">
      <string>Cadastro</string>
     </attribute>
     <attribute name="toolTip">
      <string>Manutenção de Dados de Departamento</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>360</width>
        <height>207</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Nome:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="Name_Prod">
           <property name="maxLength">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Valor Do Produto</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="IMG_Prod">
           <property name="maxLength">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Imagem Do Produto</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="Valor_Prod">
           <property name="decimals">
            <number>2</number>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1000000.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Descrição</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Descr_Prod">
         <property name="maxLength">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Categoria</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="CAT_Prod">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>361</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="ID_PRods">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="ID_Prod">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabListagem">
     <attribute name="icon">
      <iconset resource="recursos.qrc">
       <normaloff>:/depto/WINPREV.BMP</normaloff>:/depto/WINPREV.BMP</iconset>
     </attribute>
     <attribute name="title">
      <string>Listagem</string>
     </attribute>
     <attribute name="toolTip">
      <string>Exibe lista de departamentos cadastrados</string>
     </attribute>
     <widget class="QTableWidget" name="gridDepto">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>261</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID_Prod</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Imagem</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descrição</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Categoria</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCadastro">
    <property name="title">
     <string>Cadastro</string>
    </property>
    <addaction name="actionConex_o"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuListagem">
    <property name="title">
     <string>Listagem</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Sair</string>
    </property>
   </widget>
   <addaction name="menuCadastro"/>
   <addaction name="menuListagem"/>
   <addaction name="menuSobre"/>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Inicio"/>
   <addaction name="action_Anterior"/>
   <addaction name="action_Proximo"/>
   <addaction name="action_Fim"/>
   <addaction name="separator"/>
   <addaction name="action_Buscar"/>
   <addaction name="action_Novo"/>
   <addaction name="separator"/>
   <addaction name="action_Editar"/>
   <addaction name="action_Salvar"/>
   <addaction name="action_Cancelar"/>
   <addaction name="action_Excluir"/>
   <addaction name="separator"/>
   <addaction name="action_Sair"/>
  </widget>
  <action name="actionConex_o">
   <property name="text">
    <string>Conexão</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="action_Novo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/Add.bmp</normaloff>:/depto/Add.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Novo</string>
   </property>
   <property name="toolTip">
    <string>Incluir novo registro</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_Editar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/WINNEXT.BMP</normaloff>:/depto/WINNEXT.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Editar</string>
   </property>
   <property name="toolTip">
    <string>Altera dados do registro em exibição</string>
   </property>
  </action>
  <action name="action_Salvar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/Save.bmp</normaloff>:/depto/Save.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
   <property name="toolTip">
    <string>Grava o registro atual no banco de dados</string>
   </property>
  </action>
  <action name="action_Excluir">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/Minus.bmp</normaloff>:/depto/Minus.bmp</iconset>
   </property>
   <property name="text">
    <string>E&amp;xcluir</string>
   </property>
   <property name="toolTip">
    <string>Exclui o registro cujo código foi digitado</string>
   </property>
  </action>
  <action name="action_Cancelar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/UNDO.BMP</normaloff>:/depto/UNDO.BMP</iconset>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
   <property name="toolTip">
    <string>Cancela a operação atualmente em execução</string>
   </property>
  </action>
  <action name="action_Buscar">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/Oeil2.bmp</normaloff>:/depto/Oeil2.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Buscar</string>
   </property>
   <property name="toolTip">
    <string>Busca o registro cujo código foi digitado e o exibe</string>
   </property>
  </action>
  <action name="action_Inicio">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/first.bmp</normaloff>:/depto/first.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Início</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no primeiro registro e o exibe</string>
   </property>
  </action>
  <action name="action_Anterior">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/prior.bmp</normaloff>:/depto/prior.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Anterior</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no registro anterior ao atualmente exibido</string>
   </property>
  </action>
  <action name="action_Proximo">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/next.bmp</normaloff>:/depto/next.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Próximo</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no registro seguinte ao atual</string>
   </property>
  </action>
  <action name="action_Fim">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/last.bmp</normaloff>:/depto/last.bmp</iconset>
   </property>
   <property name="text">
    <string>&amp;Fim</string>
   </property>
   <property name="toolTip">
    <string>Posiciona visualização no último registro e o exibe</string>
   </property>
  </action>
  <action name="action_Sair">
   <property name="icon">
    <iconset resource="recursos.qrc">
     <normaloff>:/depto/CLOSE1.BMP</normaloff>:/depto/CLOSE1.BMP</iconset>
   </property>
   <property name="text">
    <string>Sai&amp;r</string>
   </property>
   <property name="toolTip">
    <string>Fecha a conexão, salva dados e fecha o formulário</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
